build/pandoc/defaults/html.yaml

```diff
@@ -2,23 +2,30 @@
 # Load on top of common defaults.
 to: html5
 output-file: output/manuscript.html
-# include-before-body:
+include-before-body:
+  # theme
+  - build/themes/default.html
+
```

Comment:
slight preference to remove the blank lines in the YAML list.

Also probably best to avoid adding indentation to the list items to keep the diff simple and avoid user confusion (we've found users often mess up indentation when writing YAML.

Reply:
Blank lines were to clearly separate the sections of first and third party plugins. Without them, the commented out plugins also look like their own sections:
![image](https://user-images.githubusercontent.com/8326331/106941634-162afe80-66f1-11eb-99ea-ca1c6932a56e.png)

The indentation was just how Prettier formatted it I think. It seems [the recommended way](https://www.google.com/search?q=yaml+list+indentation) is to indent with 2 spaces. Nevertheless I've updated it to revert the indentation.

Reply:
> Blank lines were to clearly separate the sections of first and third party plugins. Without them, the commented out plugins also look like their own sections:

Got it. The revised `###` make the sections clear to me, so I have the preference for removing blank lines.